concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -0,0 +1,481 @@
+Gauchat-1925-480-140	140	is better	est meilleur	meliōre	48		
+Gauchat-1925-480-141	141	than the other	que l’autre	alteru	48		
+Gauchat-1925-480-142	142	It has one	Il a une	habet	48		
+Gauchat-1925-480-143	143	width	largeur	largu + ōre	48		
+Gauchat-1925-480-144	144	double	double	dŭpla	48		
+Gauchat-1925-480-145	145	The sewing	La couture	co(n)sutūra	50		
+Gauchat-1925-480-146	146	is straight	est droite	dirēcta	50	1404	STRAIGHT
+Gauchat-1925-480-147	147	The vehicle	Le char	carru	50	2549	VEHICLE
+Gauchat-1925-480-148	148	is light	est léger	leviariu	50	1052	LIGHT (WEIGHT)
+Gauchat-1925-480-149	149	The chain	La chaîne	catēna	50	1627	CHAIN
+Gauchat-1925-480-150	150	(is) light (feminine)	(est) légère	leviaria	50		
+Gauchat-1925-480-151	151	The rope	La corde	cŏrda	52	1218	ROPE
+Gauchat-1925-480-152	152	new (feminine)	neuve	nŏva	52		
+Gauchat-1925-480-153	153	is better (feminine)	est meilleure	meliōre	52		
+Gauchat-1925-480-154	154	The circle	Le cercle	cĭrculu	52	1467	CIRCLE
+Gauchat-1925-480-155	155	is round	est rond	rotŭndu	52	1395	ROUND
+Gauchat-1925-480-156	156	The wheel	La roue	rŏta	52	710	WHEEL
+Gauchat-1925-480-157	157	(is) round (feminine)	(est) ronde	rotŭnda	54		
+Gauchat-1925-480-158	158	One must	Il faudra	fallere + habet	54		
+Gauchat-1925-480-159	159	soon	bientôt	bĕne tŏstu	54	703	SOON
+Gauchat-1925-480-160	160	mow	tondre	tŏndere	54	190	MOW
+Gauchat-1925-480-161	161	the ewes	les brebis	fētas	54	1345	EWE
+Gauchat-1925-480-162	162	sharpen (?)	racérer	reaciariare	54		
+Gauchat-1925-480-163	163	the axe	la hache	happia (+ itta)	56	677	AXE
+Gauchat-1925-480-164	164	beat in the barn	battre en grange	excŭtere	56		
+Gauchat-1925-480-165	165	grind	moudre	mŏlere	56	1033	GRIND
+Gauchat-1925-480-166	166	the wheat	le blé	blatu	56	1077	WHEAT
+Gauchat-1925-480-167	167	keep	garder	wardare	56	1477	KEEP
+Gauchat-1925-480-168	168	the livestock	les bêtes	bĕstias	56	631	LIVESTOCK
+Gauchat-1925-480-169	169	turn	tourner	(re)virare	58	1588	TURN
+Gauchat-1925-480-170	170	the earth	la terre	tĕrra	58	1228	EARTH (SOIL)
+Gauchat-1925-480-171	171	to sow	semer	seminare	58	748	SOW SEEDS
+Gauchat-1925-480-172	172	(the) hemp	(le) chanvre	cannapu	58	2041	HEMP
+Gauchat-1925-480-173	173	the beans	les fèves	fabas	58	832	BEAN
+Gauchat-1925-480-174	174	The horse	Le cheval	caballu	58	615	HORSE
+Gauchat-1925-480-175	175	is sold	est vendu	vendūtu	60		
+Gauchat-1925-480-176	176	The cow	La vache	vacca	60	1007	COW
+Gauchat-1925-480-177	177	(is) sold (feminine)	(est) vendue	vendūta	60		
+Gauchat-1925-480-178	178	The goats	Les chèvres	capras	60	1502	GOAT
+Gauchat-1925-480-179	179	(are) sold	(sont) vendues	vendūtas	60		
+Gauchat-1925-480-180	180	It is expensive	Il est cher	caru	60		
+Gauchat-1925-480-181	181	(she is) expensive	(Elle est) chère	cara	62		
+Gauchat-1925-480-182	182	(they are) expensive	(elles sont) chères	caras	62		
+Gauchat-1925-480-183	183	The pig	Le porc	pŏrcu	62	1337	PIG
+Gauchat-1925-480-184	184	is fat	est gros	grŏssu	62	2449	FAT (OF MEAT AND FOOD)
```

Comment:
This is the state of 'being big' I think?

Reply:
It's the sentence "The pig is fat", so yes, it is not really "fat of food", but rather "fat (obese)". Obelix is also called "gros" in the comics.